bindata/manifests/switchdev-config/files/configure-switchdev.sh.yaml

```diff
@@ -6,18 +6,42 @@ contents:
     #!/bin/bash
     set -eux
     input="/etc/switchdev.conf"
+    UDEV_RULE_FILE='/etc/udev/rules.d/10-persistent-net.rules'
+
     if [ ! -f $input ]; then
       echo "File /etc/switchdev.conf not exist."
       exit
     fi
 
+    append_to_file(){
+      if ! test -f "$2"
```

Comment:
can this be made more readable ?  please assign meaningful local vars to $1 $2

Reply:
updated